src/style.css

```
@@ -1,3 +1,85 @@
+html {
+  --header-height: 60px;
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  padding-top: 10px;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav {
+  font-size: 12px;
+  padding-right: 50px;
+}
+
+.logo_img {
+  padding-left: 50px;
+}
+
+.nav_links {
+  display: flex;
+  list-style: none;
+}
+
+.nav_link:nth-child(-n + 7) {
+  margin-right: 20px;
+}
+
+.nav_link a {
```

```suggestion
.nav__link {
```
style by classes instead by tags.
set .nav__link class for `a` tag